### ISSUE TITLE
Bookshelf view, + button, strange focus on narrator

### ISSUE THREAD

**Issue by llemeurfr** [edrlab/thorium-reader#575]
This problem was there before on any browser: a very small square at the top left of the svg button was handling the action. It is still true on Windows, with Narrator active. 

Note: Check first if it is about Narrator or simply Windows.

**Comment by panaC** [comment]
the reason is the input tag is reduce to 1*1 px. the actual display is the label with "+" svg with a "for"link to input action.

Talked with Laurent, we can classify this issue.

In the future, a possible resolve will be to used the electron input modal and not the input html tag with own behaviour

**Comment by panaC** [comment]
may i close the issue ?

**Comment by danielweck** [comment]
I think it is fair to close this issue at this juncture, due to lack of details, and Narrator evolutions in Windows.
